sherlock/resources/data.json

```diff
@@ -1,2218 +1,2241 @@
-    "errorType": "message",
-    "url": "https://php.ru/forum/members/?username={}",
-    "urlMain": "https://php.ru/forum/",
-    "username_claimed": "apple",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "pikabu": {
-    "errorType": "status_code",
-    "url": "https://pikabu.ru/@{}",
-    "urlMain": "https://pikabu.ru/",
-    "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "pr0gramm": {
-    "errorType": "status_code",
-    "url": "https://pr0gramm.com/user/{}",
-    "urlMain": "https://pr0gramm.com/",
-    "urlProbe": "https://pr0gramm.com/api/profile/info?name={}",
-    "username_claimed": "cha0s",
-    "username_unclaimed": "noonewouldeverusethis123123123123123123"
-  },
-  "prog.hu": {
-    "errorType": "response_url",
-    "errorUrl": "https://prog.hu/azonosito/info/{}",
-    "url": "https://prog.hu/azonosito/info/{}",
-    "urlMain": "https://prog.hu/",
-    "username_claimed": "Sting",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "radio_echo_msk": {
-    "errorType": "status_code",
-    "url": "https://echo.msk.ru/users/{}",
-    "urlMain": "https://echo.msk.ru/",
-    "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "satsisRU": {
-    "errorType": "status_code",
-    "url": "https://satsis.info/user/{}",
-    "urlMain": "https://satsis.info/",
-    "username_claimed": "red",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "social.tchncs.de": {
-    "errorType": "status_code",
-    "url": "https://social.tchncs.de/@{}",
-    "urlMain": "https://social.tchncs.de/",
-    "username_claimed": "Milan",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "spletnik": {
-    "errorType": "status_code",
-    "url": "https://spletnik.ru/user/{}",
-    "urlMain": "https://spletnik.ru/",
-    "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "svidbook": {
-    "errorType": "status_code",
-    "url": "https://www.svidbook.ru/user/{}",
-    "urlMain": "https://www.svidbook.ru/",
-    "username_claimed": "green",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "toster": {
-    "errorType": "status_code",
-    "url": "https://www.toster.ru/user/{}/answers",
-    "urlMain": "https://www.toster.ru/",
-    "username_claimed": "adam",
-    "username_unclaimed": "noonewouldeverusethis7"
-  },
-  "uid": {
-    "errorType": "status_code",
-    "url": "http://uid.me/{}",
-    "urlMain": "https://uid.me/",
-    "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
-  }
-}
\ No newline at end of file
+{

```

Comment:
You have changed line endings into `\r\n` in whole file

Reply:
I didn't get that. Can you please elaborate it a bit?

Reply:
Sure, https://en.wikipedia.org/wiki/Newline#Representation.

I suppose that you have accidentally changed new lines style from `\n` (unix style) to `\r\n` (windows style) in `data.json` file. 

That is indicated by overall changes count (~top right corner of website):  
- you have inserted +2,547 lines
- you have removed −2,512 lines
And in `Files Changed` tab you will see that every line of that files has been changed.

Reply:
Thanks for explaining it in detail.

Reply:
@pkubiak  Is the format okay now? I have also added the support of YourQuotes.